appveyor.yml

```diff
@@ -0,0 +1,31 @@
+before_build:
+  - cmd: dotnet restore 
+build_script:
+  - cmd: dotnet build 
+test_script:
+  - cmd: dotnet test 
+test: on
+os: Visual Studio 2019
+deploy:
+  release: Centaurus $(APPVEYOR_REPO_TAG_NAME)
+  provider: GitHub
+  auth_token:
+    secure: jnMwGuWtmzFm6Zz0hL+TNL3cU/LlIdzklvxGExt6hBiiZZKu6rWjERl1KWqfU8z8
```

Comment:
Is this ok? Are you sure?? It might be called "secure" for a reason.

Reply:
Yes, it's encrypted.